codeship-steps.yml

```diff
@@ -86,6 +89,8 @@
               command: './packages/devops/scripts/ci/deployBackend.sh psss-server'
             - name: deploy-report-server
               command: './packages/devops/scripts/ci/deployBackend.sh report-server'
+            - name: deploy-server-boilerplate
+              command: './packages/devops/scripts/ci/deployBackend.sh server-boilerplate'
```

Comment:
Is server-boilerplate a package that needs deploying? I thought it was just an internal dependency? (it will still get built when we build the internal dependencies even if it's not listed here)

Reply:
ah yes of course